From 90b6f5dfcb9c9f41dcbd59b006ea703f2f78b3b1 Mon Sep 17 00:00:00 2001
From: Hans de Goede <hdegoede@redhat.com>
Date: Tue, 25 Mar 2014 11:48:58 +0100
Subject: [PATCH 1/3] Drop $RAWCPPFLAGS when generating startx

startx.cpp contains things like #if defined(__SCO__), and
$RAWCPPFLAGS contains -undef causing these to not get set.

Signed-off-by: Hans de Goede <hdegoede@redhat.com>
---
 cpprules.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cpprules.in b/cpprules.in
index 0931bee..781676a 100644
--- a/cpprules.in
+++ b/cpprules.in
@@ -15,4 +15,4 @@ CPP_SED_MAGIC = $(SED) -e '/^\#  *[0-9][0-9]*  *.*$$/d' \
 SUFFIXES = .cpp
 
 .cpp:
-	$(AM_V_GEN)$(RAWCPP) $(RAWCPPFLAGS) $(CPP_FILES_FLAGS) < $< | $(CPP_SED_MAGIC) > $@
+	$(AM_V_GEN)$(RAWCPP) $(CPP_FILES_FLAGS) $< | $(CPP_SED_MAGIC) > $@
-- 
1.9.0

